_ont_xgs/ont-hisense-ltf7267-bha+.md

```diff
@@ -0,0 +1,395 @@
+             |--Eth   :0x0301
+GEM: 1025  | (BI, TCONT:0x101, allocId:1281)
+           --Bridge + 802.1p:0x1
+           --Map Pri 0x17
+           --Ani Map Vlan Total Num 1 Vid 100
+             |--Eth   :0x0301
+```
+
+## Check VLAN filter 
+
+```sh
+Cortina> enable
+Cortina# config
+Cortina(config)# omci
+Cortina(config-omci)# show stream
+```
+
+You will see the following output:
+
+```sh
+Link to PPTP eth: ------->instance 0x301
+    input TPID 0x8100 output TPID 0x8100
+    dsMode:Inverse us rules
+    -------------------------------------------------------------------
+    |               Filter              |          Treatment          |
+    |-----------------------------------|-----------------------------|
+    |    Outer   |         Inner        | R |   Outer    |    Inner   |
+    |------------|----------------------| E |------------|------------|
+    | PRI | VID  | PRI | VID  | Filter  | M | PRI | VID  | PRI | VID  |
+    |-----|------|-----|------|---------|---|-----|------|-----|------|
+    |  15 | 4096 |  14 | 4096 |No filter| 3 |  15 |    0 |  15 |    0 |
+    |  14 | 4096 |  14 | 4096 |No filter| 3 |  15 |    0 |  15 |    0 |
+    |  15 | 4096 |  15 | 4096 |No filter| 0 |  15 | 4096 |   0 |    1 |
+    |  15 | 4096 |   8 |  835 |No filter| 1 |  15 |    0 |   8 |  100 |
+    |-----|------|-----|------|---------|---|-----|------|-----|------|
+```
+
+## Changing the Hardware Version
+
+```sh
+# vi /config/scfg.txt
+```
+
+Append line below to the file and save it to change HWVER
+
+```
+CHAR-ARRAY      CFG_ID_GPON_VERSION = {0x46,0x35,0x36,0x38, 0x34,0x53,0x5f,0x76, 0x31,0x00,0x00,0x00, 0x00,0x00}; ##GPON version string, default value is V1.0
+```
+
+Reboot ONT to apply the change
+
+## Changing the Equipment ID
+
+```sh
+# vi /config/scfg.txt
+```
+
+Append line below to the file and save it to change EQID
+
+```
+CHAR-ARRAY      CFG_ID_GPON_EQID = {0x46,0x49,0x42,0x45, 0x52,0x20,0x42,0x6f, 0x78,0x00,0x00,0x00, 0x00,0x00,0x00,0x00, 0x00,0x00,0x00,0x00}; ##GPON ME ONU2G equiment id value, default is saturn
+```
+
+Reboot ONT to apply the change
+
+## Changing Serial Number
+
+```sh
+# vi /config/scfg.txt
+```
+
+Append lines below to the file and save it to change Serial Number
+
+```
+STRING CFG_ID_PON_VENDOR_ID = SMBS;
+INT CFG_ID_PON_VSSN = 0xAABBCCDD;
+```
+
+Reboot ONT to apply the change
+
+## Changing OLT Emulation Type
+
+```sh
+# vi /config/scfg.txt
+```
+
+Append line below to the file and save it to change Serial Number
+
+```
+STRING          CFG_ID_PON_OLT_TYPE                             = ALCL; ##GPON OLT Vendor name, support ZTE,ADTRAN,ALCL,CALIX,SUMITOMO,CORTINA,HUAWEI
+```
+
+Reboot ONT to apply the change
+
+## Changing OMCC Version
+
+{% include alert.html content="In Italy, if you are under Huawei OLT it's mandatory to use 0xA3, while on Alcatel 0xB4, otherwise you will get O5 status but no MIBs - Note that this can be quirk of TIM Italy" alert="Warning" icon="svg-warning" color="red" %}
```

Comment:
```suggestion
{% include alert.html content="In Italy, if you are under some Huawei OLT it's mandatory to use 0xA3, while on Alcatel 0xB4, otherwise you will get O5 status but no MIBs - Note that this can be quirk of TIM Italy" alert="Warning" icon="svg-warning" color="red" %}
```